libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/GenState.hs

```diff
@@ -887,7 +892,8 @@ genPlutusScript proof tag = do
       corescript = case proof of
         Alonzo _ -> script
         Babbage _ -> script
-        _ -> error ("Only Alonzo and Babbage have PlutusScripts. " ++ show proof ++ " does not.")
+        Conway _ -> script
+        _ -> error ("Only Alonzo, Babbage, and Conway have PlutusScripts. " ++ show proof ++ " does not.")
```

Comment:
maybe we could keep this from rotting by saying "PlutusScripts are available starting in the Alonzo era".

Reply:
done